releasenotes/notes/features-88f5ec5ec6b5c380.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    Enabling the option to strip the arguments of the commands collected by the process Agent on Windows.
```

Comment:
The option previously existed, I would call this more of a bugfix to account for spaces in the executable path on Windows